style.css

```diff
@@ -468,3 +502,242 @@ body {
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+@media screen and (min-width: 768px) {
+  header {
+    margin: 1.25rem 2.5rem;
+  }
+
+  .logo {
+    margin-left: 155px;
+  }
+
+  p {
+    font-family: 'Poppins', sans-serif;
+    text-align: left;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .homepage {
+    background: url(./images/Aboutme-Big-screen.png) no-repeat;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 12px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    padding: 359px 260px;
+  }
+
+  .intro {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .intro h1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    text-align: left;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    width: 100%;
+  }
+
+  .detail {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    padding: 0;
+  }
+
+  .connect {
+    padding: 0;
+  }
+
+  .connect h3 {
+    display: flex;
+    flex-direction: row;
+    align-items: start;
+    justify-content: left;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .logos ul {
+    display: flex;
+    gap: 20px;
+    color: #505f79;
+    list-style: none;
+    margin-left: 0.1rem;
+    margin-top: 0.5rem;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .heading-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+    gap: 32px;
+    font-family: 'Poppins', sans-serif;
+    color: #634634;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-right: 155px;
+  }
+
+  .menubars {
+    display: none;
+  }
+
+  .cards {
+    display: grid;
+    padding: 16px;
+    grid-template-columns: repeat(2, 1fr);
+    justify-content: center;
+    place-items: center;
+    gap: 20px;
+    background-color: #e5e5e5;
+  }
```

Comment:
- [ ] Please align the cards according to the figma design. Your cards seem like this on 768px screens: 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/91486791/f9721607-dcb0-4692-925f-feba4936b119)
figma: 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/91486791/e55ad3d7-31f9-4271-9abd-b6b7714abbe1)